packages/pyright-internal/src/version.json

```diff
@@ -0,0 +1 @@
+"1.7.0"
```

Comment:
Lets see a cli test case that checks the version output

Reply:
sounds like a pain in the ass, the code for it is just 2 console.logs, very straightforward and unlikely to break imo